infra/proposer.go

```diff
@@ -46,8 +62,8 @@ type Proposer struct {
 	logger *log.Logger
 }
 
-func CreateProposer(addr string, crypto *Crypto, logger *log.Logger) *Proposer {
-	endorser, err := CreateEndorserClient(addr, crypto.TLSCACerts)
+func CreateProposer(addr string, tls [][]byte, logger *log.Logger) *Proposer {
```

Comment:
nit - avoid var name `tls`, which is vague, and overlapping with pkg name

Reply:
will change to `TLSCACert` or any over suggestion?